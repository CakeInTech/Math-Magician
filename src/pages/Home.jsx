import React from 'react';
import css from './home.module.css';

const Home = () => (
  <div className={css.container}>
    <h1>Welcome to our page!</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. In eaque illo et asperiores temporibus tempore ad aperiam at fuga distinctio labore quasi maiores laudantium odio consectetur eveniet earum consequatur, dignissimos, deleniti eos minus.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Et, distinctio mollitia! Recusandae magni odit odio earum in ad unde, accusantium quae, nihil ullam reiciendis ut harum, tenetur eos hic mollitia exercitationem rerum numquam iusto eius.
    </p>
  </div>
);

export default Home;