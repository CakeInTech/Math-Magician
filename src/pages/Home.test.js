import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders correctly', () => {
  const linker = renderer.create(<Home />).toJSON();
  expect(linker).toMatchSnapshot();
});
