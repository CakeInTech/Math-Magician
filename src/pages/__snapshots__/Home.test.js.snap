// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
  data-testid="home"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. In eaque illo et asperiores temporibus tempore ad aperiam at fuga distinctio labore quasi maiores laudantium odio consectetur eveniet earum consequatur, dignissimos, deleniti eos minus.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. In eaque illo et asperiores temporibus tempore ad aperiam at fuga distinctio labore quasi maiores laudantium odio consectetur eveniet earum consequatur, dignissimos, deleniti eos minus.
  </p>
</div>
`;
